Gopkg.toml

```diff
@@ -3,7 +3,14 @@
 
 required = [
   "github.com/tektoncd/plumbing/scripts",
+  "github.com/knative/test-infra/scripts",
   "github.com/knative/test-infra/tools/dep-collector",
+  "k8s.io/gengo/examples/defaulter-gen/generators",
+  "k8s.io/gengo/args",
+  "k8s.io/gengo/generator",
+  "k8s.io/gengo/namer",
+  "k8s.io/gengo/types",
+  "k8s.io/gengo/examples/deepcopy-gen/generators",
```

Comment:
I admittedly do not know very much about k8s code generation. But, I think the `pipeline` repo and `hack/update-codegen.sh` use the `https://github.com/kubernetes/code-generator` library instead of `k8s.io/gengo`. What's the difference between these two, and should we be using `code-generator` instead of `gengo` to be more consistent with the `pipeline` repo?

Reply:
`k8s.io/gengo` is used by `github.com/kubernetes/code-generator` internally.

Reply:
Ah, thank you 😅 

I think we need to include `code-generator` in the `Gopkg.toml` file because `update-codegen.sh` uses it. Here's a line from the build-tests output (https://tekton-releases.appspot.com/build/tekton-prow/pr-logs/pull/tektoncd_triggers/17/pull-tekton-triggers-build-tests/1147135672893575168/):
> /go/src/github.com/tektoncd/triggers/hack/update-codegen.sh: line 29: ../../../k8s.io/code-generator/generate-groups.sh: No such file or directory

Reply:
Will do, ty

Reply:
vendored, ty